post/index.md

```diff
@@ -0,0 +1,295 @@
+                                        // March
+                                        _eq: 3,
+                                    },
+                                },
+                            ],
+                        },
+                        {
+                            // Year
+                            _or: [
+                                {
+                                    "year(published_date)": {
+                                        _eq: 2023,
+                                    },
+                                },
+                                {
+                                    "year(published_date)": {
+                                        _eq: 2022,
+                                    },
+                                },
+                            ],
+                        },
+                        {
+                            // Title search
+                            title: {
+                                _contains: "gpt",
+                            },
+                        },
+                    ],
+                },
+            })
+        )
+
+//Response
+[{...},{...}] // total count 2
+```
+
+## Aggregation & Grouping
+
+### Get Filter Count By Year
+
+```js
+// Get articles count by year for showing in filters
+const result = await client.request(
+	aggregate('articles', {
+		aggregate: { count: '*' },
+        groupBy: ['year(published_date)']
+	})
+);
+
+// Response
+{
+  "data": [
+    {
+      "published_date_year": 2022,
+      "count": 35
+    },
+    {
+      "published_date_year": 2023,
+      "count": 44
+    }
+  ]
+}
+```
+
+![Filter Year Count](./images/filter-year-count.jpeg)
+
+### Get Filter Count By Month
+
+```js
+// Get articles count by month for showing in filters
+const result = await client.request(
+	aggregate('articles', {
+		aggregate: { count: '*' },
+        groupBy: ['month(published_date)']
+	})
+);
+
+// Response
+{
+  "data": [
+    {
+      "published_date_month": 1,
+      "count": 4
+    },
+    {
+      "published_date_month": 2,
+      "count": 16
+    },
+    {
+      "published_date_month": 3,
+      "count": 15
+    },
+    {
+      "published_date_month": 4,
+      "count": 9
+    },
+    {
+      "published_date_month": 5,
+      "count": 8
+    },
+    {
+      "published_date_month": 6,
+      "count": 4
+    },
+    {
+      "published_date_month": 7,
+      "count": 2
+    },
+    {
+      "published_date_month": 8,
+      "count": 4
+    },
+    {
+      "published_date_month": 9,
+      "count": 2
+    },
+    {
+      "published_date_month": 10,
+      "count": 5
+    },
+    {
+      "published_date_month": 11,
+      "count": 6
+    },
+    {
+      "published_date_month": 12,
+      "count": 6
+    }
+  ]
+}
+```
+
+![Month Count](./images/month-filter-count.jpeg)
+
+-   Below image shows few more options
+    ![Article List!](./images/all-filters-options.png)
```

Comment:
Let's remove this. It's repetitive. We've already demonstrated month and year are interchangable logic-wise.